CHANGELOG.md

```diff
@@ -2,6 +2,48 @@
 
 All notable changes to this project will be documented in this file.
 
+## 2021-07-03
+* Added new VFX 2022 Draft images with the following versions:
+    * python-3.9.5
+    * tbb-2020_U3
+    * boost-1.75.0
+    * cppunit-1.15.1
+    * log4cplus-1.1.2
+    * glew-2.1.0
+    * glfw-3.1.2
+    * qt-5.15.2
+    * pyside-5.15.2
+    * cmake-3.20.5
+    * openexr-2.5.5
```

Comment:
2022 should be imath 3.x, openexr 3.x

Reply:
thanks indeed! I think my subconscious made me avoid the upgrade to avoid having to deal with the asymmetric requirements, as I won't have an Imath package for the previous years... I'll work on that now :)

Reply:
Actually I "just" need to split Imath from OpenEXR for previous years to match the current structure...